track_oracle/example/track_reader_example.cxx

```diff
@@ -27,7 +27,9 @@
 #include <track_oracle/file_formats/file_format_schema.h>
 #include <track_oracle/file_formats/file_format_manager.h>
 #include <track_oracle/file_formats/file_format_base.h>
+#if KWIVER_ENABLE_KPF
```

Comment:
I assume that KWIVER_ENABLE_KPF is passed as a definition to the build step.

Reply:
It seems so... The same symbol is used in `file_format_manager.cxx`, although I don't actually see how it gets defined. (Someone whose kwiver build enables KPF should be able to trivially verify if it is being defined or not. IIRC, I have it `OFF` due to [Fedora#1509421](https://bugzilla.redhat.com/show_bug.cgi?id=1509421).)

If I turn on CMake `KWIVER_ENABLE_KPF`, the build falls apart, with one of the errors being:

```
In file included from .../track_oracle/example/track_reader_example.cxx:31:0:
.../track_oracle/file_formats/kpf_utils/kpf_utils.h:48:10: fatal error: track_oracle/file_formats/kpf_utils/kpf_utils_export.h: No such file or directory
 #include <track_oracle/file_formats/kpf_utils/kpf_utils_export.h>
```

...which is the same error I get without the patch, so it does appear to get defined... somehow...